elm_frontend/src/Papers/OtherPathway.elm

```diff
@@ -0,0 +1,44 @@
+module Papers.OtherPathway exposing (OtherPathwayPaper, view, viewList)
+
+import GeneralTypes exposing (PaperMetadata, renderPaperMetaData)
+import Html exposing (Html, div, h2, p, section, text)
+import Html.Attributes exposing (class)
+
+
+
+-- TYPES
```

Comment:
We have these types comments in some files like this one where there is only one type declaration and view things below it. How do you feel about removing the comments?